nuxeo-common/src/main/java/org/nuxeo/common/utils/SizeUtils.java

```diff
@@ -44,7 +44,7 @@ private SizeUtils() {
      * @return the size
      * @throws NumberFormatException if the size cannot be parsed
      */
-    public static long parseSizeInBytes(String string) throws NumberFormatException {
+    public static long parseSizeInBytes(String string) {
```

Comment:
I know some tools suggest to remove the `throws NumberFormatException` because it's a runtime exception, but this is wrong, it's part of the contract of the method so should be there.